docs/index.md

```diff
@@ -130,3 +130,43 @@ napari is extensible! Find plugins, or develop your own!
 :::
 
 ::::
+
+## funding
+
+If you like napari and want to support our [mission](mission-and-values),
+please consider a [donation](https://numfocus.org/donate-to-napari) to our
+efforts.
```

Comment:
Draga mentioned that donations are still being match by CZI, perhaps we should mention that here? it's sort of an incentive isn't it?